_rules/video-description-track-f196ce.md

```diff
@@ -112,8 +112,8 @@ A video element without audio.
 
 ```html
 <video controls>
-	<source src="../test-assets/rabbit-video/silent.mp4" type="video/mp4" />
-	<source src="../test-assets/rabbit-video/silent.webm" type="video/webm" />
+	<source src="../test-assets/rabbit-video/silent.mp4" type="video/mp4">
+	<source src="../test-assets/rabbit-video/silent.webm" type="video/webm">
 	<track
```

Comment:
`<track>` is also a void element, so it shouldn't have the end slash, similar to `<source>`